opencog/atoms/proto/CMakeLists.txt

```diff
@@ -5,7 +5,10 @@ OPENCOG_ADD_ATOM_TYPES(atom_types.script
   atom_types.inheritance
   core_types.scm
   core_types.pyx)
-ADD_CUSTOM_TARGET(opencog_atom_types DEPENDS atom_types.h)
+
+ADD_CUSTOM_TARGET(opencog_atom_types DEPENDS atom_types.h
```

Comment:
this is overkill, since all five files will change, when the types.script file changes.

Reply:
Yes, I was going to write comment on it but then thought that it is simpler add all files as dependencies.